ucm2/Intel/sof-hda-dsp/sof-hda-dsp.conf

```diff
@@ -115,3 +126,51 @@ If.Capture {
 		cset "name='Capture Switch' on"
 	]
 }
+
+Include.hdmi-pcm.File "/common/pcm/hdmi.conf"
+
+If.Hdmi3-iec61937 {
```

Comment:
I would suggest create a macro for this If and call it for requested Device/Index combo.

Reply:
Hrm, OK

Reply:
When I thought that I understand how UCM works I just cannot get this macro to work.
It looks like that if I use `Device` and `Index` as parameters for the new macro then it will conflict with HdmiPCM's Device and Index? If I use different name then I get no match it looks like, at least nothing is saved.

I would love to do this as a followup ;)

Reply:
I got this error if I use Device/Index parameter: `unable to evaluate '${evali:$__Device}'`
```
DefineMacro.HdmiIsIec61937.If.iec61937 {
	Condition {
		Type RegexMatch
		Regex "((^|,)[${var:__Device}](,|$))"
		String "${var:Iec61937Pcms1}"
	}
	True {
		Macro.hdmi.HdmiPCM { Device "${var:__Device}" Index "${var:__Index}" }
		Define.hdmi_iec61937 "yes"
	}
}

Macro.0.HdmiIsIec61937 { Device 3 Index 0 }
Macro.1.HdmiIsIec61937 { Device 4 Index 1 }
Macro.2.HdmiIsIec61937 { Device 5 Index 2 }
```

Reply:
My guess is that a Macro parameter cannot be a variable expression?

Reply:
Yes, the arguments were not substituted. I added this feature to `Syntax 7` (refer to alsa-lib commits bellow). So it was omitted in implementation.